src/pybind/mgr/dashboard/controllers/rbd.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 # pylint: disable=unused-argument
-# pylint: disable=too-many-statements,too-many-branches
+# pylint: disable=too-many-statements,too-many-branches,too-many-nested-blocks
```

Comment:
I'm wondering if we can remove ``too-many-statements`` as we already deactivated it in ``.pylintrc``?

```suggestion
# pylint: disable=too-many-branches,too-many-nested-blocks
```

Would it also be possible to merge line 2 and 3?